sdv/sequential/par.py

```diff
@@ -70,19 +70,20 @@ class PARSynthesizer(LossValuesMixin, BaseSynthesizer):
     _model_sdtype_transformers = {'categorical': None, 'numerical': None, 'boolean': None}
 
     def _get_context_metadata(self):
+        context_metadata_dict = {}
         context_columns_dict = {}
         context_columns = self.context_columns.copy() if self.context_columns else []
         if self._sequence_key:
             context_columns += self._sequence_key
+            context_metadata_dict['primary_key'] = self._sequence_key[0]
```

Comment:
The [docs](https://docs.sdv.dev/sdv/reference/metadata-spec/metadata-api#set_sequence_key) make it seem like we only support one sequence key

Reply:
Yeah, PAR stores the sequence key in a list as a relic from when we used to support composite sequence keys. I think we decided to just disallow more than one sequence key and didn't bother refactoring in case we wanted to allow more than one sequence key again in the future. 

Since we only allow one sequence key though, we know that if there's a sequence key it'll just be the first (and only) item in the list.